src/pages/PMT_PaymentWizard.page

```diff
@@ -1,155 +1,363 @@
+            </div>
+        </apex:outputText>
         
-        <!-- Create payment schedule -->
-        <apex:pageBlock title="{!$Label.pmtWizardCreateSchedule}" rendered="{!AND(NOT(isClosedLost), OR(haveAmount, removePaidPayments))}">
-            <apex:pageBlockSection id="pmtGrid" columns="1">
-                <apex:pageBlockSectionItem >       
-                    <apex:outputLabel value="{!$Label.pmtWizardNumberOfPayments}" />
-                    <apex:selectList value="{!numberofPayments}" id="paymentCount" multiselect="false" size="1">
-                        <apex:selectOptions value="{!items}"/>
-                    </apex:selectList>
-                </apex:pageBlockSectionitem>
-            
-                <apex:pageBlockSectionItem >       
-                    <apex:outputLabel value="{!$Label.pmtWizardFirstPaymentDate}"/>
-                    <apex:inputField value="{!samplePayment.npe01__Scheduled_Date__c}"/>
-                </apex:pageBlockSectionItem>
-            
-                <apex:pageBlockSectionItem >       
-                    <apex:outputLabel value="{!$Label.pmtWizardInterval}" />
-                    <apex:outputpanel >
-                        <apex:selectList value="{!interval}" id="intervals" multiselect="false" size="1">
-                            <apex:selectOptions value="{!intervals}"/>
-                        </apex:selectList>
-                
-                        <apex:selectList value="{!intervalunit}" id="intervalunits" multiselect="false" size="1">
-                            <apex:selectOptions value="{!intervalunits}"/>
-                        </apex:selectList>
-                    </apex:outputpanel>
-                </apex:pageBlockSectionItem> 
-    
-                <apex:pageBlockSectionItem >       
-                    <apex:outputLabel value="{!$ObjectType.npe01__OppPayment__c.Fields.npe01__Payment_Method__c.Label}"/>
-                    <apex:inputField value="{!samplePayment.npe01__Payment_Method__c}"/>
-                </apex:pageBlockSectionItem>
-            </apex:pageblocksection>
-            <div style="text-align: center;">
-            <apex:commandButton action="{!calculate}" value="{!$Label.pmtWizardBtnCalcPayments}"/></div>
-        </apex:pageBlock>
+        <!-- No unpaid amount warning -->
+        <apex:outputText rendered="{!AND(NOT(ISNULL(CurrentOpp)),NOT(haveAmount), NOT(isClosedLost))}">
+            <div class="slds-notify slds-notify--alert slds-theme--inverse-text slds-theme--alert-texture" role="alert">
+                <p>
+                    {!$Label.pmtWizardWarning}
+                </p>
+                <p>
+                    {!$Label.pmtWizardNoUnpaidAmount}&nbsp;&nbsp;
+                    <a href="{!URLFOR($Action.Opportunity.View, currentOpp.id)}">
+                        {!currentOpp.Name}
+                    </a>
+                </p>
+            </div>
+            <div class="slds-grid slds-grid--align-center">
```

Comment:
This button bumps up against the alert and the div below it.  It would probably look better if this div had some padding (slds-p-vertical--medium maybe?)